ajax/libs/react-flip-move/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-flip-move",
+  "filename": "react-flip-move.min.js",
+  "homepage": "http://joshwcomeau.github.io/react-flip-move/examples",
+  "version": "2.9.17",
+  "description": "Effortless animation between DOM changes (eg. list reordering) using the FLIP technique.",
+  "author": "Joshua Comeau <joshwcomeau@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/joshwcomeau/react-flip-move.git"
+  },
+  "keywords": [
+    "react",
+    "react-dom",
+    "animation",
+    "magic-move",
+    "component",
+    "react-component",
+    "flip",
+    "web-animations"
+  ],
+  "npmName": "react-flip-move",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
Let's use `**/!(*.gz)` to avoid `react-flip-move.min.js.gz` in old versions (v2.x).